src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+document.querySelector('#submit').addEventListener('click', function(event) {
+  event.preventDefault();
+}, false);
```

Comment:
if you are using js for form, you can clear inputs after submitting in js
The main thing - **clear inputs after form submission **